upoi/map/models.py

```diff
@@ -1,3 +1,70 @@
 from django.db import models
 
-# Create your models here.
+
+class User(models.Model):
```

Comment:
ok, then i renamed that model to UpoiUser.

class UpoiUser(models.Model):
    user = models.OneToOneField(User,
        on_delete=models.CASCADE,
        )